lib/modules/betteReddit.js

```diff
@@ -219,8 +219,12 @@ module.go = () => {
 function commentsLinksNewTabs(ele = document.body) {
 	const links = ele.querySelectorAll('.thing div.md a');
 	links::forEachChunked(link => {
-		link.target = '_blank';
-		link.rel = 'noopener noreferer';
+		const { baseURI, href } = link;
+
+		if (!(href.includes(baseURI) && (href.includes('#!settings') || href.includes('#res:settings')))) {
```

Comment:
The `href.includes(...)` checks are already written for you, so you can `import * as SettingsNavigation from './settingsNavigation';` at the top of this file (in alphabetical order please) and replace the `href.includes...` part with `SettingsNavigation.isSettingsUrl(href)`.